libkode/printer.cpp

```diff
@@ -956,16 +951,47 @@ void Printer::printImplementation( const File &file, bool createHeaderInclude )
     filename.prepend( d->mOutputDirectory + '/' );
 
   QFile implementation( filename );
-  if ( !implementation.open( QIODevice::WriteOnly ) ) {
-    qWarning( "Can't open '%s' for writing.", qPrintable( filename ) );
-    return;
+  printCodeIntoFile(out, &implementation);
+}
+
+void Printer::printCodeIntoFile(const Code &code, QFile *file)
+{
+  QString outText = code.text();
+  bool identical = true;
+  QTextStream fileStream( file );
```

Comment:
Should be created inside the following if(), next to the other QTextStream. It's not needed here.

Reply:
Many thanks for the review, I will fix the mentioned issues soon.

The fileStream is also used in the if (!identical) { if below:
https://github.com/martonmiklos/kode/blob/skip_owerwrite_if_output_identical/libkode/printer.cpp#L987